mobify/platform-scaffold
# Rename plp and pdp

to `ProductList` and `ProductDetails`, like in the PWA.

~~**JIRA**: (link to JIRA ticket)~~

## Changes
- Rename plp and pdp
- Fix some lint errors
- Ensure lint checking happens in CI

## How to test-drive this PR
- Run `npm test`
- Run `npm run dev`
- Check out http://localhost:3000
